pyro/infer/reparam/discrete_cosine.py

```diff
@@ -31,6 +31,8 @@ class DiscreteCosineReparam(UnitJacobianReparam):
         noise; when -1 this transforms violet noise to white noise; etc. Any
         real number is allowed. https://en.wikipedia.org/wiki/Colors_of_noise.
     """
-    def __init__(self, dim=-1, smooth=0.):
+    def __init__(self, dim=-1, smooth=0., *,
+                 experimental_allow_batch=False):
```

Comment:
does the new flag need a docstring?

Reply:
no, I think it is fine undocumented because it is experimental. This kwarg simply controls whether the experimental new behavior is allowed.